indico/modules/events/registration/templates/display/registration_summary.html

```diff
@@ -4,60 +4,76 @@
 {% set class_mapping = {'complete': 'accept', 'pending': 'warning', 'unpaid': 'warning'} %}
 
 <div class="action-box {{ class_mapping[registration.state.name] }}">
-   <div class="section">
```

Comment:
I'd fix the indentation separately... it doesn't help the readability of the PR :/